tests/unit/useOnyxTest.ts

```diff
@@ -102,7 +102,7 @@ describe('useOnyx', () => {
 
             const {result} = renderHook(() => useOnyx(ONYXKEYS.TEST_KEY));
 
-            expect(result.current[0]).toEqual(null);
+            expect(result.current[0]).toEqual(undefined);
```

Comment:
You can use this method instead of those `toEqual`.
```suggestion
            expect(result.current[0]). toBeUndefined;
```